iree/compiler/Dialect/HAL/IR/BUILD

```diff
@@ -33,6 +33,7 @@ cc_library(
         "HALOpFolders.cpp",
         "HALOpInterface.cpp.inc",
         "HALOps.cpp",
+        "HALOps.cpp.inc",
```

Comment:
This is already present in `textual_hdrs` below (https://docs.bazel.build/versions/2.0.0/be/c-cpp.html#cc_library.textual_hdrs).This file does not compile on its own though Bazel, without further changes. I tested moving from `textual_hdrs` to `srcs` and internal (Google) builds, which use fairly strict checks, failed:

```
HALDialect.cpp:66:10: error: use of private header from outside its module: 'third_party/iree/iree/compiler/Dialect/HAL/IR/HALOps.cpp.inc'
```

Was there a particular reason why you wanted to add this line, or did it just appear to be missing?

Reply:
I simply missed that it is included in `textual_hdrs`. Sorry for the noise. Within CMake something like `textual_hdrs` does not exist (yet?) and it appeared missing when comparing the Bazel and some autogenerated CMake configuration.